src/payment/strategies/klarna/klarna-credit.ts

```diff
@@ -27,3 +27,19 @@ export interface KlarnaAuthorizationResponse {
         invalid_fields: string[];
     };
 }
+
+export interface KlarnaUpdateSessionParams {
```

Comment:
This doesn't seem right, considering the actual definition of the method, all these properties seem optional. Which looks like the reason we have the `| {}` in the above definition.

A more accurate type description could be:
```ts
export default interface KlarnaCredit {  
	// ...
	load(params: KlarnaLoadParams, callback: (res: KlarnaLoadResponse) => void): void; 
	load(params: KlarnaLoadParams, data: KlarnaUpdateSessionParams, callback: (res: => void): void; 
	// ...
}

export type KlarnaUpdateSessionParams = Partial<{
    billing_address: KlarnaAddress;
    shipping_address: KlarnaAddress;
}>
```